packages/database/src/migrations/20200925064855-ReorderLaosSchoolsOverlayGroups-modifies-data.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+import { codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const LAOS_SCHOOLS_OVERLAY_GROUP_ORDERS = [
+  'School_Indicators_EiE',
+  'Laos_Schools_School_Indicators_EiE_Sub_National_Group',
+  'Laos_Schools_Textbook_student_Ratio_Group',
+  'Laos_Schools_Student_Numbers_Group',
+  'Laos_Schools_Population_Group',
+  'Laos_Schools_Repetition_Rates_Group',
+  'Laos_Schools_Drop_Out_Rates_Group',
+  'Development_Partner',
+];
+
+const WORLD_MAP_OVERLAY_GROUP_CODE = 'World_Group';
```

Comment:
Reminder that this will need to change based on https://github.com/beyondessential/tupaia/pull/1388#discussion_r495644912